cmd/gitops/get/profiles/cmd.go

```diff
@@ -0,0 +1,58 @@
+package profiles
```

Comment:
Unfortunately I think we should remove the `gitops get profiles` sub command from the PR :(. My bad being a bit slow and not co-ordinating! Sorry about that. The work is being done w/ the new gitops based profiles in #1227

Reply:
That's great 🙂 I will remove this part 🎇